force-app/main/default/classes/tasks/CollectionsTasks.cls

```diff
@@ -26,6 +41,28 @@ public class CollectionsTasks {
     Integer sum,
     List<Integer> listWithElementsToCheck
   ) {
+
+    List<Integer> sortedList = new List<Integer>(listWithElementsToCheck);
```

Comment:
I checked this implementation on the datasets from the previous tasks and it failed. Took me a while to understand the trick though :)

This implementation looks like a hack for the test data - @baksale is working on the new set of test data to make this hack unavailable. Nice try though :)